tests/components/risco/test_binary_sensor.py

```diff
@@ -7,23 +7,30 @@
 from homeassistant.helpers import device_registry as dr, entity_registry as er
 from homeassistant.helpers.entity_component import async_update_entity
 
-from .util import TEST_CONFIG, TEST_SITE_UUID, setup_risco
-from .util import two_zone_alarm  # noqa: F401
+from .util import (  # noqa: F401
+    TEST_CLOUD_CONFIG,
+    TEST_LOCAL_CONFIG,
+    TEST_SITE_UUID,
+    setup_risco_cloud,
+    setup_risco_local,
+    two_zone_cloud,
+    two_zone_local,
```

Comment:
Don't import pytest fixtures. Either define them in the same module if they are only used for the tests of that module, or in a `conftest.py` module in the package if they are used by more than one module. The fixtures are then available by default to the tests below in the tree.